src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/tts_is_ready_request.cc

```diff
@@ -50,7 +50,12 @@ TTSIsReadyRequest::TTSIsReadyRequest(
                    rpc_service,
                    hmi_capabilities,
                    policy_handler)
-    , EventObserver(application_manager.event_dispatcher()) {}
+    , EventObserver(application_manager.event_dispatcher()) {
+  related_capabilities_requests_ = {
```

Comment:
@LitvinenkoIra can we give a better name? Let it be long but at least something like: 
`requests_required_for_capabilities` - if it has such meaning